assets/css/documentation.css

```diff
@@ -70,3 +70,7 @@ section {
 .margin-left--half {
   margin-left: 0.5rem;
 }
+
+.line-break {
+  margin-bottom: 24px;
```

Comment:
Please change to 15px

Reply:
@shahednasser Thanks for the feedback. I have made the change.